### PR TITLE
Create NavBar

### LINKED ISSUE #22: [1pt] Create empty My profile - Setup (group task)
- Create a route and a view component. Use `<NavLink />` for the page navigation links and style active class to indicate which section/page the user is currently on (underline active navigation link).
- This view should be empty - you will add content in separate tasks.

### LINKED ISSUE #23: [1pt] Create website header with empty nav - Setup (group task)
- Download the [free image](https://www.flaticon.com/free-icon/planet_3212567?term=space&page=1&position=19&page=1&position=19&related_id=3212567&origin=style) for the app logo.
- Create **empty** navigation (this will be filled in in separate tasks).

### LINKED ISSUE #24: [1pt] Create app and install basic dependencies - Setup (group task)
- Create React App.
- Install [React Redux](https://react-redux.js.org/), [Redux Logger](https://www.npmjs.com/package/redux-logger) and [React Router](https://reactrouter.com/web/guides/quick-start).

### PR DESCRIPTION
**Pull request by Boku52500**
- Initialized project with react/redux.
- Installed all the required and needed dependencies.
- Created NavBar with routers.
- Made NavBar functional.

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    <!--
+      This HTML file is a template.
+      If you open it directly in the browser, you will see an empty page.
+
+      You can add webfonts, meta tags, or analytics to this file.
+      The build step will place the bundled scripts into the <body> tag.
+
+      To begin the development, run `npm start` or `yarn start`.
+      To create a production bundle, use `npm run build` or `yarn build`.
+    -->
```
Comment (sebalp100) [inline public/index.html]: Kindly remove unnecessary comments. Removing these kinds of comments will make your code look more professional and also is going to save you some lines of code.

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+const root = ReactDOM.createRoot(document.getElementById('root'));
+root.render(
+  <Router>
+    <App />
+  </Router>,
+);
+
+// If you want to start measuring performance in your app, pass a function
+// to log results (for example: reportWebVitals(console.log))
+// or send to an analytics endpoint. Learn more: https://bit.ly/CRA-vitals
```
Comment (sebalp100) [inline src/index.js]: Kindly remove unnecessary comments. Removing these kinds of comments will make your code look more professional and also is going to save you some lines of code.

**Review by sebalp100 — CHANGES_REQUESTED** [review]
### **STATUS: CHANGES REQUIRED ♻️** 

Hi, @Boku52500  good job so far following the instructions. However, there are still some requirements missing from your project. You can check the comments at the bottom for more info.

dog


**HIGHLIGHTS:**

- Good job using the correct ES6 syntax 💯 
- Branch name is descriptive 💯 
- Commits are descriptive 💯 
- Linters are passing all the tests 💯
- NavBar is working as expected

**REQUIRED CHANGES ♻️  :**

Great job so far, however, there are still some changes to be made before you can continue.

_For more information on your required changes, please check below ⬇️_

File: src/index.js (hunk 2 above)
Comment (Boku52500) [reply src/index.js]: @sebalp100 Thank you for reviewing my code and leaving me notes. I have removed all the comments.

**Review by sebalp100 — APPROVED** [review]
STATUS: APPROVED 🟢

ok


Highlights
✔️ Descriptive PR
✔️ Passing linters
✔️ Good branch name
✔️ Functionality working as expected

Great implementation of all the requirements in this milestone. 👏

You have done an amazing job 👍

Your project is complete! There is nothing else to say other than... it's time to merge it. :shipit:

Congratulations! 🎉

CHEERS AND HAPPY CODING!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**